src/test/java/teammates/test/driver/FieldValidator.java

```diff
@@ -0,0 +1,42 @@
+package teammates.test.driver;
+
+import teammates.common.util.Assumption;
+import teammates.common.util.SanitizationHelper;
+
+/**
+ * Used to handle the data validation aspect for test.
```

Comment:
`Holds additional methods for {@link teammates.common.util.FieldValidator} used only in tests.`